app/src/organisms/Devices/ProtocolRun/__tests__/BackToTopButton.test.tsx

```diff
@@ -102,19 +102,18 @@ describe('BackToTopButton', () => {
     })
   })
 
-  it('should be disabled with modules not connected tooltip when there are missing moduleIds', () => {
+  it('should not disabled with modules not connected tooltip when there are missing moduleIds', () => {
     when(mockUseUnmatchedModulesForProtocol)
       .calledWith(ROBOT_NAME, RUN_ID)
       .mockReturnValue({
         missingModuleIds: ['temperatureModuleV1'],
         remainingAttachedModules: [],
       })
-    const { getByRole, getByText } = render()
+    const { getByRole } = render()
     const button = getByRole('button', { name: 'Back to top' })
-    expect(button).toBeDisabled()
-    getByText('Make sure all modules are connected before proceeding to run')
+    expect(button).not.toBeDisabled()
   })
-  it('should be disabled with modules not connected and calibration not completed tooltip if missing cal and moduleIds', async () => {
+  it('should not be disabled with modules not connected and calibration not completed tooltip if missing cal and moduleIds', async () => {
```

Comment:
oh also lets delete the mention of tooltip in the test case names since we no longer render a tooltip
```suggestion
  it('should not be disabled if missing cal and moduleIds', async () => {
```